policyengine_us/reforms/reconciliation/reconciled_cdcc.py

```diff
@@ -0,0 +1,71 @@
+from policyengine_us.model_api import *
+from policyengine_core.periods import period as period_
+from policyengine_core.periods import instant
+
+
+def create_reconciled_cdcc() -> Reform:
+    class cdcc_rate(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "CDCC credit rate"
+        unit = USD
+        definition_period = YEAR
+        reference = "https://www.law.cornell.edu/uscode/text/26/21#a_2"
+
+        def formula(tax_unit, period, parameters):
+            p = parameters(period).gov.irs.credits.cdcc
+            agi = tax_unit("adjusted_gross_income", period)
+
+            # First phase-out
+            excess_agi = max_(0, agi - p.phase_out.start)
+            increments = np.ceil(excess_agi / p.phase_out.increment)
+            percentage_reduction = increments * p.phase_out.rate
+            phased_out_rate = max_(
+                p.phase_out.min, p.phase_out.max - percentage_reduction
+            )
+
+            # Second phase-out
+            p_ref = parameters(period).gov.contrib.reconciliation.cdcc
+            filing_status = tax_unit("filing_status", period)
+            second_excess_agi = max_(
```

Comment:
use below, do same for increment
```suggestion
            second_phase_out_start = p_ref.phase_out.second_start[filing_status]
            second_excess_agi = max_(
```